crates/bevy_render/src/texture/image.rs

```diff
@@ -531,6 +531,38 @@ impl Image {
         image
     }
 
+    /// A transparent white 1x1x1 image.
+    ///
+    /// Contrast to [`Image::default`], which is opaque.
+    pub fn transparent() -> Image {
+        // We rely on the default texture format being RGBA8UnormSrgb
+        // when constructing a transparent color from bytes.
+        // If this changes, this function will need to be updated.
```

Comment:
How will we find out? Will it fail to compile?

Reply:
This will panic when the examples are run :)